modules/delete-kn-trigger.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_content-type: PROCEDURE
 [id="delete-kn-trigger_{context}"]
-= Deleting a trigger by using the Knative CLI
+= Deleting a trigger by using the Knative (kn) CLI
```

Comment:
same comment throughout, don't think this is necessary in titles, just in main content

Reply:
@rh-max